src/i18n.js

```diff
@@ -0,0 +1,21 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import HttpApi from 'i18next-http-backend';
+
+i18n
+  .use(initReactI18next)
+  .use(LanguageDetector)
+  .use(HttpApi)
+  .init({
+    fallbackLng: 'en',
+    detection: {
+      order: ['htmlTag', 'cookie', 'localStorage', 'path', 'subdomain'],
+      caches: ['cookie'],
+    },
```

Comment:
In here somewhere, we need a "resources" key which will hold our translation files. Please check the documentation to do that.